src/gui/properties/piecesbar.cpp

```diff
@@ -286,7 +286,10 @@ void PiecesBar::showToolTip(const QHelpEvent *e)
     }
     else {
         stream << simpleToolTipText();
-        stream << '\n' << tr("Hold Shift key for detailed information");
+        if (showDetailedInformation) // metadata are not available at this point
+            stream << '\n' << tr("Wait until metadata become available to see detailed information");
```

Comment:
>Why do we show the progress bar when there is no metadata available? 

I find it weird that progress bar is sometimes showing, sometimes not, sometimes popping up after random time...
IMO its better to have a uniform look.